app/views/shared/partials/_dates_form.html.erb

```diff
@@ -50,3 +50,10 @@
     <% end %>
   <% end %>
 </div>
+
+<script type="text/javascript">
+  $('#construction_dates_0__date_start').change(function() {
+    alert('Change');
+    $('#construction_dates_0__date_end').val($('#construction_dates_0__date_start').val())
+  });
```

Comment:
warum gibts das?

![Bildschirmfoto 2021-07-08 um 12 31 01](https://user-images.githubusercontent.com/1942953/124907400-667ff280-dfe8-11eb-9fe4-e02408225d0c.png)

is das ganze javascript hier nur testing?

wenn start date 5.6.2021 ist und enddate 23.12.2021 und jemand dann das start date ändert, soll das end date auf selbigen wert gesetzt werden? das war eine anforderung @theresastone @pwilimzig?

Reply:
Der Alert muss auf jeden Fall noch raus, das war nur zum Debugging. 

Die Anforderung war, dass per Default Start und Enddatum gleich sind. Wenn also das Startdatum gesetzt wird, soll das Enddatum per default gleich sein und kann im Nachtrag dann noch abgeändert werden. Jetzt muss @pwilimzig sagen, ob das so okay ist, oder ob er es sich anders gedacht hat.

Reply:
done in [e96d3f](https://github.com/ikuseiGmbH/smart-village-app-cms/commit/e96d3f1b57b3c22e92ae37e1a7737ea5eeb0e0ac)